src/module/system/text-editor.ts

```diff
@@ -466,7 +466,7 @@ class TextEditorPF2e extends TextEditor {
                 element.appendChild(document.createTextNode(" "));
 
                 const details = document.createElement("span");
-                if (dc && showDC && Number.isNumeric(dc)) {
+                if (dc && showDC && Number.isNumeric(dc) && statistic) {
```

Comment:
This conditional branch is a little absurd at this point and should be refactored.

Reply:
I was following the pattern of the following "else if" statement, which also contained unnecessary duplicate checks